### PR TITLE
Start from Vault activation

### PR DESCRIPTION
**Pull request by banteg**
Requires Laika release of Vaults or later.

### CODE REVIEW

File: contracts/GuestList.vy [hunk 1]
```diff
@@ -177,31 +175,52 @@ def enough_yfi(user: address, threshold: uint256) -> bool:
...
-    assert msg.sender == self.bouncer
+    assert msg.sender == self.bouncer  # dev: unauthorized
     self.bouncer = new_bouncer
 
 
 @external
 def bribe_the_bouncer(guest: address = msg.sender):
     """
-    Sneak into the party by bribing the bouncer with 2% of the entrance cost.
+    Sneak into the party by bribing the bouncer. The pass works for all future vaults.
```
Comment (fubuloubu) [inline contracts/GuestList.vy]: Would say "this works for any vault that uses this guest list"

As we may change in the future

File: contracts/GuestList.vy [hunk 2]
```diff
@@ -26,6 +26,7 @@ struct ProtectedLiquidity:
 interface Vault:
     def balanceOf(user: address) -> uint256: view
     def getPricePerFullShare() -> uint256: view
```
Comment (fubuloubu) [inline contracts/GuestList.vy:28]: This is gonna be fun when the new Vault changes this endpoint 😬

File: contracts/GuestList.vy [hunk 3]
```diff
@@ -69,10 +70,7 @@ bancor: Bancor
 @external
 def __init__():
     self.bouncer = msg.sender
-    self.activation = block.timestamp
-    # constants
-    self.min_bag = 10 ** 18  # 1 YFI to enter
-    self.ape_out = 30 * 86400  # 30 days falloff
+    self.bribe_cost = 25 * 10 ** 15  # 0.025 YFI life pass
```
Comment (Macarse) [inline contracts/GuestList.vy:73]: you can also do `25 * 1e15`

**Review by Macarse — APPROVED** [review]

File: contracts/GuestList.vy (hunk 3 above)
Comment (banteg) [reply contracts/GuestList.vy:73]: Vyper says "Cannot perform multiplication between dislike types"

File: contracts/GuestList.vy (hunk 2 above)
Comment (banteg) [reply contracts/GuestList.vy:28]: This is only used with the old yYFI vault.

**Pull request merged**